### DISCUSSION TITLE
Release date for sponsored Brands and display for version 3 reporting

### DISCUSSION THREAD

**Discussion by Vignesh-Siva96** [amzn/ads-advanced-tools-docs#70] [Q&A] [answered]
Is there any tentative date or timeline for the version 3 brands and display reporting?

**Comment by ncoughlin** [comment]
I asked this question also and they said no timeline.

**Reply by Vignesh-Siva96** [reply, marked as answer]
@ncoughlin,

I have got a response from the support. The estimated go live date for the Sponsored Brands V3 reports might be around July and there is no ETA for Sponsored display V3 reports as of now.
